Include dist-info for python3-libdnf

evan-goode opened the pull request:
This is the libdnf version of this patch [0] for DNF 4.

Per [1], Python packages should include some minimal distribution metadata, e.g. to make the package appear in `pip list`.

[0] https://github.com/rpm-software-management/dnf/pull/1997
[1] https://packaging.python.org/en/latest/specifications/core-metadata/#core-metadata

= changelog =
msg: Add metadata for `libdnf` Python package
type: bugfix

evan-goode committed:
Include dist-info for python3-libdnf

This is the libdnf version of this patch [0] for DNF 4.

Per [1], Python packages should include some minimal distribution
metadata, e.g. to make the package appear in `pip list`.

[0] https://github.com/rpm-software-management/dnf/pull/1997
[1] https://packaging.python.org/en/latest/specifications/core-metadata/#core-metadata

= changelog =
msg: Add metadata for `libdnf` Python package
type: bugfix

jan-kolarik reviewed (approved):
LGTM, thanks!

jan-kolarik commented:
Hmm, I've noticed there is not your username connected with the commit. Do you want to modify that? The same thing is with the already pushed dnf PR ...
![image](https://github.com/rpm-software-management/libdnf/assets/110612307/3ab08961-4e03-40ca-ba00-a086708eb1f0)
In the `git log` there is `Author: Your Name <you@example.com>`.

evan-goode commented:
Oh thanks, something wiped out my git config. Just fixed it. I noticed that after the dnf PR was merged but I thought I had fixed it before committing this.

I'm guessing it would not be easy to fix the issue in dnf?  It is still the last commit so perhaps a force push would not disrupt anyone's work.

jan-kolarik commented:
> Oh thanks, something wiped out my git config. Just fixed it. I noticed that after the dnf PR was merged but I thought I had fixed it before committing this.
> 
> I'm guessing it would not be easy to fix the issue in dnf? It is still the last commit so perhaps a force push would not disrupt anyone's work.

Hmm, looking into some comments in [this SO question](https://stackoverflow.com/questions/8480365/changing-git-authorship-after-git-push), I'd rather keep the remote as it is :slightly_smiling_face:

evan-goode commented:
Alright, I think we can live with it.
